icons/qappmenu-top-left.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+
<!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
+<svg fill="#000000" width="800px" height="800px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+
<g data-name="Layer 2">
+
<g data-name="diagonal-arrow-left-up">
+
<rect width="24" height="24" transform="rotate(90 12 12)" opacity="0"/>
+
<path d="M17.71 16.29L9.42 8H15a1 1 0 0 0 0-2H7.05a1 1 0 0 0-1 1v8a1 1 0 0 0 1 1H7a1 1 0 0 0 1-1V9.45l8.26 8.26a1 1 0 0 0 1.42 0 1 1 0 0 0 .03-1.42z"/>
+
</g>
+
</g>
+
</svg>
```

Comment:
Nit: Please use LF instead of CRLF and add a trailing newline.

Reply:
All arrow svgs are redone in Inkscape with CC0 Public Domain